probe/endpoint/dns_snooper.go

```diff
@@ -1,4 +1,5 @@
-// +build linux,amd64 ppc64le
+// +build linux
+// +build amd64 ppc64le
```

Comment:
Not sure why this change is causing a test-failure, as the command "COVERDIR=./coverage make RM= CODECGEN_UID=23 tests" working fine for me on my setup. 
Any pointers would really help.

Reply:
Could resolve this failure, by latest commit.